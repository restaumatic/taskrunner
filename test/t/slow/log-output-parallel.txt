# check logs/foo.log logs/bar.log
taskrunner -n foo sh -c '
  echo "foo standard output"
  sleep 0.1
  echo "foo standard error" >&2
' &
taskrunner -n bar sh -c '
  echo "bar standard output"
  sleep 0.1
  echo "bar standard error" >&2
' &
wait
