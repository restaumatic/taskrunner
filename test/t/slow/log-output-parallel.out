-- logs/foo.log:
stdout | foo standard output
stderr | foo standard error
-- logs/bar.log:
stdout | bar standard output
stderr | bar standard error
